zk/stages/stage_sequence_execute.go

```diff
@@ -93,7 +93,7 @@ type ExecuteBlockCfg struct {
 	zk        *ethconfig.Zk
 }
 
-func StageExecuteBlocksCfg(
+func StageSequenceBlocksCfg(
```

Comment:
Should this be `StageSequenceExecutionCfg` instead with the returned type of `ExecuteBlockCfg` renamed to match also?

Reply:
renamed